### PR TITLE
Update to Outdated files in the dev-1.17-ko.7 branch

### PR DESCRIPTION
**Pull request by ysyukr**
### 54 of 58 files to be modified
- [x] content/ko/_index.html
- [x] content/ko/docs/concepts/architecture/nodes.md
- [x] content/ko/docs/concepts/configuration/assign-pod-node.md
- [x] content/ko/docs/concepts/containers/runtime-class.md
- [x] content/ko/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md
- [x] content/ko/docs/concepts/overview/kubernetes-api.md
- [x] content/ko/docs/concepts/overview/what-is-kubernetes.md
- [x] content/ko/docs/concepts/overview/working-with-objects/kubernetes-objects.md
- [x] content/ko/docs/concepts/services-networking/connect-applications-service.md
- [ ] content/ko/docs/concepts/services-networking/dual-stack.md
> #19376 에서 반영됨
- [x] content/ko/docs/concepts/services-networking/endpoint-slices.md
- [x] content/ko/docs/concepts/services-networking/ingress.md
- [x] content/ko/docs/concepts/services-networking/service.md
- [x] content/ko/docs/concepts/storage/volume-pvc-datasource.md
- [x] content/ko/docs/concepts/storage/volumes.md
- [x] content/ko/docs/concepts/workloads/controllers/cron-jobs.md
- [x] content/ko/docs/concepts/workloads/controllers/daemonset.md
- [x] content/ko/docs/concepts/workloads/controllers/deployment.md
- [x] content/ko/docs/concepts/workloads/controllers/jobs-run-to-completion.md
- [x] content/ko/docs/concepts/workloads/controllers/replicaset.md
- [x] content/ko/docs/concepts/workloads/controllers/replicationcontroller.md
- [x] content/ko/docs/concepts/workloads/controllers/statefulset.md
- [x] content/ko/docs/concepts/workloads/pods/pod.md
- [x] content/ko/docs/home/_index.md
- [x] content/ko/docs/reference/glossary/annotation.md
- [x] content/ko/docs/reference/glossary/cluster.md
- [x] content/ko/docs/reference/glossary/container-env-variables.md
- [x] content/ko/docs/reference/glossary/deployment.md
- [x] content/ko/docs/reference/glossary/image.md
- [ ] content/ko/docs/reference/glossary/ingress.md
> 번역시 차이가 없어 미반영
- [x] content/ko/docs/reference/glossary/init-container.md
- [x] content/ko/docs/reference/glossary/kube-proxy.md
- [x] content/ko/docs/reference/glossary/kube-scheduler.md
> #18092 에서 변경시 일부 잘못 변경된 것으로 판단되어 #19708 PR 제출
- [x] content/ko/docs/reference/glossary/kubelet.md
- [x] content/ko/docs/reference/glossary/replication-controller.md
- [x] content/ko/docs/reference/glossary/selector.md
- [x] content/ko/docs/reference/glossary/taint.md
- [x] content/ko/docs/reference/glossary/volume.md
- [x] content/ko/docs/reference/kubectl/cheatsheet.md
- [x] content/ko/docs/reference/using-api/api-overview.md
- [x] content/ko/docs/reference/using-api/client-libraries.md
- [x] content/ko/docs/setup/_index.md
- [x] content/ko/docs/setup/best-practices/multiple-zones.md
- [x] content/ko/docs/setup/learning-environment/minikube.md
- [x] content/ko/docs/setup/production-environment/tools/kops.md
- [x] content/ko/docs/setup/production-environment/windows/user-guide-windows-containers.md
- [x] content/ko/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md
- [ ] content/ko/docs/tasks/access-application-cluster/web-ui-dashboard.md
> 번역시 차이가 없어 미반영
- [x] content/ko/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md
- [x] content/ko/docs/tasks/run-application/horizontal-pod-autoscale.md
- [x] content/ko/docs/tasks/tools/install-minikube.md
- [x] content/ko/docs/tutorials/_index.md
- [x] content/ko/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html
- [x] content/ko/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html
- [x] content/ko/docs/tutorials/kubernetes-basics/expose/expose-intro.html
- [x] content/ko/docs/tutorials/services/source-ip.md
- [x] content/ko/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md
- [ ] content/ko/docs/tutorials/stateless-application/guestbook.md
> 번역시 차이가 없어 미반영

### 4 files to be deleted
- [x] content/ko/docs/concepts/cluster-administration/federation.md
- [x] content/ko/docs/tutorials/online-training/_index.md
- [x] content/ko/docs/tutorials/online-training/overview.md
- [x] content/ko/includes/federated-task-tutorial-prereqs.md

from #19662 
/language ko

### CODE REVIEW

File: content/ko/docs/home/_index.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ menu:
     weight: 20
     post: >
       <p>개념, 튜토리얼 및 참조 문서와 함께 쿠버네티스 사용하는 방법을 익힐 수 있다. 또한, <a href="/editdocs/" data-auto-burger-exclude>문서에 기여하는 것도 도움을 줄 수 있다</a>!</p>
+description: >
+  쿠버네티스는 컨테이너화된 애플리케이션의 배포, 확장 및 관리를 자동화하기위한 오픈소스 컨테이너 오케스트레이션 엔진이다. 오픈소스 프로젝트는 Cloud Native Computing Foundation에서 주관한다.
```
Comment (ClaudiaJKang) [inline content/ko/docs/home/_index.md]: > origin
>   Kubernetes is an open source container orchestration engine for automating deployment, scaling, and management of containerized applications. The open source project is hosted by the Cloud Native Computing Foundation.

```suggestion
  쿠버네티스는 컨테이너화된 애플리케이션의 배포, 확장 및 관리를 자동화하기 위한 오픈소스 컨테이너 오케스트레이션 엔진이다. 오픈소스 프로젝트는 Cloud Native Computing Foundation에서 주관한다.
```

- [ ] 띄어쓰기가 필요합니다.

**Comment by ysyukr** [comment]
@ClaudiaJKang 
주신 의견 확인해서 업데이트 진행했습니다. :)

**Comment by ClaudiaJKang** [comment]
@ysyukr 감사합니다. 
/lgtm

**Comment by gochist** [comment]
@ysyukr 감사합니다. 

`content/ko/docs/concepts/overview/kubernetes-api.md` 끝 부분의 충돌을 해결하고 충돌 부분에 있던 일부 쿠버네티스 API 오브젝트 이름을 외래어 표기법에 따라 옮기는 수정을 추가한 후에 승인합니다.

/lgtm
/approve

**Pull request merged**